src/lib/fonts-css.ts

```diff
@@ -268,6 +267,9 @@ const fontList: FontDisplay[] = [
     },
 ];
 
+const assetsUrl = (path: string): string =>
+    `//assets.guim.co.uk/static/frontend/${path}`;
```

Comment:
I think we should just use `https` rather than protocol-less

Reply:
I copied this from `src/lib/assets.ts`, but I'm happy to change if we prefer specifying `https`

Reply:
I believe nowadays it's an anti-pattern and browsers are starting to upgrade all requests by default to `https` so makes sense to just hard code it here. 👍